src/visualBuilder/index.ts

```diff
@@ -276,6 +284,9 @@ export class VisualBuilder {
                     VisualBuilderPostMessageEvents.GET_ALL_ENTRIES_IN_CURRENT_PAGE,
                     getEntryIdentifiersInCurrentPage
                 );
+                visualBuilderPostMessage?.send(
```

Comment:
What would happen if event listener is not register in Visual Builder?

Reply:
it would still be persisting the same behaviour current main has